### ISSUE TITLE
New post not published

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#818]
```
What's the problem?
I click start a new post, write my new post and then click publish my post.
A message "Your post was successfully published." is also displayed.
However when I check my post , it is not there , I have tried it several 
times...



What browser are you using?


Firefox 27.0


What version of ScribeFire are you running?
4.2.4
```

-----
Original issue reported on code.google.com by `mrcou...@gmail.com` on 10 Feb 2014 at 2:20

Attachments:
* [scribefirebug.png](https://storage.googleapis.com/google-code-attachments/scribefire-chrome/issue-818/comment-0/scribefirebug.png)

**Comment by GoogleCodeExporter** [comment]
```
I am however able to edit the existing posts and publish.The problem is with 
new posts only.Thanks.
```

-----
Original comment by `mrcou...@gmail.com` on 10 Feb 2014 at 2:23

**Comment by GoogleCodeExporter** [comment]
```
This is fixed  posting in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:33
* Changed state: **Fixed**
